src/lib/netlist/plib/putil.cpp

```diff
@@ -11,25 +12,28 @@
 // needed for getenv ...
 #include <initializer_list>
 
-namespace plib
-{
-	namespace util
-	{
-		static constexpr const char PATH_SEP = compile_info::win32::value ? '\\' : '/';
-		static constexpr const char *PATH_SEPS = compile_info::win32::value ? "\\/" :"/";
+namespace plib {
+	namespace util {
+
+		static constexpr const char PATH_SEP = compile_info::win32::value ? '\\'
+																		  : '/';
+		static constexpr const char *PATH_SEPS = compile_info::win32::value
+													 ? "\\/"
+													 : "/";
```

Comment:
Now you’ve got inconsistency in how these two lines are formatted.

Reply:
This is also caused by column limitation of 80.